onnxruntime/core/graph/graph.cc

```diff
@@ -4063,6 +4066,23 @@ ONNX_NAMESPACE::GraphProto Graph::ToGraphProtoWithExternalInitializers(const std
         external_stream << raw_data[index];
       }
 
+      // update external_offset for alignment
+      if (align_offset && tensor_bytes_size > align_threshold) {
```

Comment:
what's the else case?

Reply:
There are 2 case of else:
1. user choose not to align offset
2. small initializer (whose size smaller than threshold) that we choose not to align since it doesn't have much impact on memory usage

For else case, we don't need to align offset

Reply:
Why we do not need to align offset for small initializers? How does this impact memory mapping? Do we choose to fail the mapping and read it into memory?